Doctrine/Orm/DataProvider.php

```diff
@@ -122,6 +123,16 @@ public function getCollection(ResourceInterface $resource, Request $request)
         $classMetaData = $manager->getClassMetadata($entityClass);
         $identifiers = $classMetaData->getIdentifier();
 
+        $i = 0;
```

Comment:
Nitpicking but this is not necessary as `getAssociationNames()` already returns a numerically indexed array. You can just use the key.